pkg/controller/machine.go

```diff
@@ -1036,8 +1036,91 @@ func shouldReconcileMachine(machine *v1alpha1.Machine, now time.Time) bool {
 	return true
 }
 
+// // setMachineConfigurations checks if there are any machine-controller configurations on the machine-object, and sets if there are any.
```

Comment:
Removed unused/commented block.

Reply:
Ah, thanks. Yet to take the final look. Btw, can did you make the suggestion/changes, I don't see it, here. I'll anyways remove the commented code soon.

Reply:
No. That's all it was.